adam-core/src/main/scala/org/bdgenomics/adam/converters/TranscriptEffectConverter.scala

```diff
@@ -212,9 +212,9 @@ private[adam] object TranscriptEffectConverter extends Serializable with Logging
       }
     }
 
-    val attrOpt = Option(vc.getAttributeAsString("ANN", null))
+    val attrOpt = Option(vc.getAttributeAsList("ANN"))
     try {
-      attrOpt.flatMap(attr => parseAndFilter(attr))
+      attrOpt.flatMap(attr => parseAndFilter(attr.asScala.map(_.asInstanceOf[java.lang.String])))
```

Comment:
Well, I guess we're good here then. You said you've got more tests coming, right?